WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -226,4 +226,55 @@ d_melchisedechianism_holy_order = {
 	monthly_income = 2
 
 	mercenary_type = zealots_composition
-}	
\ No newline at end of file
+}
+
+c_priscillianism_holy_order = {
+	color = { 0.5 0.7 0.5 }
+
+	graphical_culture = hashshashingfx
```

Comment:
A joke on the name "Avengers", as the Hashhashin look like supervillains and can be used for "heroes". Also, I don't know all  of the graphical cultures or if there are Iberian unique units.